data/languages/r5900.cspec

```diff
@@ -18,86 +18,82 @@
           <entry size="1" alignment="1" />
           <entry size="2" alignment="2" />
           <entry size="4" alignment="4" />
-          <entry size="8" alignment="4" />
+          <entry size="8" alignment="8" />
```

Comment:
Do they definitely have to be 8 byte aligned?

Reply:
100% positive that longs are 8 bytes and 8 byte aligned.